plugins/modules/vmware_object_role_permission_info.py

```diff
@@ -0,0 +1,247 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2018, Derek Rushing <derek.rushing@geekops.com>
+# Copyright: (c) 2018, VMware, Inc.
+# Copyright: (c) 2021, Ansible Project
+# Copyright: (c) 2021, Abhijeet Kasurde <akasurde@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+DOCUMENTATION = r"""
+---
+module: vmware_object_role_permission_info
+short_description: Gather information about object's permissions
+description: This module can be used to gather object permissions on the given VMware object.
+author:
+- Abhijeet Kasurde (@Akasurde)
+notes:
+    - Tested on ESXi 6.5, vSphere 6.7
+    - The ESXi login user must have the appropriate rights to administer permissions.
+    - Supports check mode.
+requirements:
+    - "python >= 2.7"
+    - PyVmomi
+options:
+  object_name:
+    description:
+    - The object name to assigned permission.
+    - Mutually exclusive with I(moid).
+    type: str
+  object_type:
+    description:
+    - The object type being targeted.
+    default: 'Folder'
+    choices: ['Folder', 'VirtualMachine', 'Datacenter', 'ResourcePool',
+              'Datastore', 'Network', 'HostSystem', 'ComputeResource',
+              'ClusterComputeResource', 'DistributedVirtualSwitch']
+    type: str
+  moid:
+    description:
+    - Managed object ID for the given object.
+    - Mutually exclusive with I(object_name).
+    aliases: ['object_moid']
+    type: 'str'
+extends_documentation_fragment:
+- community.vmware.vmware.documentation
+version_added: "1.9.0"
```

Comment:
The version_added needs a change because 1.9.0 is released.
What is the next release version?
1.10.0?